packages/eslint-config-base/index.js

```diff
@@ -1,8 +1,14 @@
 module.exports = {
   extends: ["plugin:prettier/recommended"],
   plugins: ["prettier", "import", "flowtype"],
+  env: {
+    browser: true,
+    commonjs: true,
+    es6: true,
+    jest: true,
+    node: true,
+  },
   parserOptions: {
-    ecmaVersion: 2018,
```

Comment:
The new env `es6` settings will set this to `6` automatically and we dont have to provide it ([eslint docs](https://eslint.org/docs/user-guide/configuring/language-options#specifying-parser-options))